src/ensembl/production/metadata/grpc/servicer.py

```diff
@@ -12,11 +12,11 @@
 from ensembl.production.metadata.grpc import ensembl_metadata_pb2_grpc
 
 from ensembl.production.metadata.grpc.utils import connect_to_db, get_species_information, get_assembly_information, \
```

Comment:
Should we think of:
- split `utils` in a proper module long files kill maintanability 
- import `*` instead?

Reply:
For the sake of clarity and avoiding naming conflicts, I implemented a third option which is import utils like so
```
import ensembl.production.metadata.grpc.utils as utils
```
and whenever a function from utils is needed it can be imported
```
conn = utils.connect_to_db()
```

Reply:
I wonder how a naming conflict could happen, but your proposal works for me.

Reply:
Naming conflict may occur using `import *` when we have two functions or classes with the same name
It may never happen, but just in case

Reply:
I know what naming conflicts means :-) I just meant this is really unlikely to happen!